### DISCUSSION TITLE
View of all sensors of ring alarm

### DISCUSSION THREAD

**Discussion by bigzbigz** [tsightler/ring-mqtt#555] [General Discussion]
Hello,
I managed to buy a ring alarm with several sensors.
I already have a docker home assistant with zigbee2mqtt and mosquitto mqtt broker that work perfectly.
I want to manage ring alarm throught home assistant and I have a question. If I use ring-mqtt-ha-addon addon is it possible to view the state of all the ring alarm sensor? For example, I plan to read ring door sensors to switch on light in my house.
Could you have an answser to my request ?
Regards.
Bigz

**Comment by tsightler** [comment]
The list of supported devices is here:  https://github.com/tsightler/ring-mqtt/wiki#supported-devices-and-features

All devices on that list will appear in HA and can be used to trigger any automations, that's pretty much the entire point of this addon.

**Reply by bigzbigz** [reply]
Thank you for the answer
